app/src/main/res/values/styles.xml

```diff
@@ -118,4 +118,10 @@
         <item name="android:paddingTop">12dp</item>
     </style>
 
+    <style name="Hedvig.Theme.InfoCard">
```

Comment:
This style does not contain the background color for warning cards, and it appears to be used for warning cards, but is named info card 🤔

Reply:
So this style is only for setting the card properties. For setting the colors we are still using the themes, like this:

```
android:theme="@style/ThemeOverlay.Hedvig.RenewalCard"
style="@style/Hedvig.Theme.InfoCard"
```

Reply:
Having a theme at the top level with `<item name="strokeWidth">1dp</item>` will apply a stroke on every child, in this case the buttons.

Reply:
Sensible.